src/jaxoplanet/experimental/starry/basis.py

```diff
@@ -17,6 +17,17 @@
 
 
 def basis(lmax):
+    """Full change of basis matrix from spherical harmonics to Green's basis
+
+    Parameters
+    ----------
+    lmax : int
+        maximum degree of the spherical harmonic basis
+
+    Returns
+    -------
+    TODO
+    """
```

Comment:
Can you update the docstrings to be in the "Google" format that is used elsewhere?

https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings